spirv_glsl.cpp

```diff
@@ -5016,6 +5039,22 @@ void CompilerGLSL::emit_instruction(const Instruction &instruction)
 			require_extension("GL_OES_standard_derivatives");
 		break;
 
+	case OpDPdxFine:
+		UFOP(dFdxFine);
```

Comment:
Hm, Indeed, can't find any ES extension for this. Probably best to throw if ES, and require that extension if desktop GLSL < 4.5.